lib/wpscan/web_site.rb

```diff
@@ -55,6 +55,9 @@ def redirection(url = nil)
     redirected_uri = URI.parse(add_trailing_slash(add_http_protocol(url)))
     if response.code == 301 || response.code == 302
       redirection = response.headers_hash['location']
+      if redirection[0] == '?'
+        redirection = "/#{redirection}"
```

Comment:
the redirection should be the same as on / so you extend the if below to smth like `if / or ?`